whisperv6/doc.go

```diff
@@ -47,6 +47,7 @@ const (
 	messagesCode           = 1   // normal whisper message
 	powRequirementCode     = 2   // PoW requirement
 	bloomFilterExCode      = 3   // bloom filter exchange
+	peerRateLimitCode      = 8   // update of the peer rate limit
```

Comment:
Why `8`? Just some number higher than 3 to avoid conflicts for some time?

Reply:
yes it was like that. but i need to remove this. as config is now advertised as a part of handshake